core/src/database/migrations.cpp

```diff
@@ -280,5 +358,27 @@ namespace ledger {
 
         }
 
+        template <> void rollback<5>(soci::session& sql) {
+            // ERC20 tokens
+            sql << "DROP TABLE erc20_tokens";
+
+            // ERC20 operations
+            sql << "DROP TABLE erc20_operations";
+
+            // ERC20 accounts
+            sql << "DROP TABLE erc20_accounts";
+
+            // ETH operations
+            sql << "DROP TABLE ethereum_operations";
+
+            // ETH transactions
+            sql << "DROP TABLE ethereum_operations";
```

Comment:
Probably you wanted to `DROP TABLE ethereum_transactions` here ?

Reply:
Argh, nice catch.